docs/development/affiliated-packages.rst

```diff
@@ -409,6 +409,30 @@ files manually`_ section since this explains what many of the files do.
 
    All other settings can stay on their default value.
 
+   If you need to mock any Python packages or C libraries that can not be
+   installed and built by ReadTheDocs, you should include the following mocking
```

Comment:
minor: it's "Read the Docs" or "readthedocs"; I don't think they use CamelCase anywhere